parser/src/main/java/com/linkedin/avroutil1/parser/avsc/AvroParseContext.java

```diff
@@ -28,7 +28,7 @@ public class AvroParseContext {
     private boolean sealed = false;
     private Map<String, AvscParseResult> knownNamedSchemas = null;
     private Map<String, List<AvscParseResult>> duplicates = null;
-    private Map<String, List<SchemaOrRef>> externalReferences = null;
+    private List<SchemaOrRef> externalReferences = null;
```

Comment:
@radai-rosenblatt `externalReferences` is not used at all. Does it have a purpose or is it something we can clean up?

Reply:
this is what we will use to lookup any still-unresolved references "elsewhere" - like dependency jars